flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/functions/SqlLikeUtils.java

```diff
@@ -58,6 +59,22 @@ public static boolean like(String s, String pattern, String escape) {
         return Pattern.matches(regex, s);
     }
 
+    /** SQL {@code ILIKE} function. */
+    public static boolean ilike(String s, String patternStr) {
+        final String regex = sqlToRegexLike(patternStr, null);
```

Comment:
this can be replaced by `ilike(String s, String patternStr, null)`

Reply:
Thanks. It's a good advice. i will fix this.